ethereum/contracts/query/QueryResponse.sol

```diff
@@ -56,6 +80,9 @@ abstract contract QueryResponse {
     bytes public constant responsePrefix = bytes("query_response_0000000000000000000|");
     uint8 public constant VERSION = 1;
     uint8 public constant QT_ETH_CALL = 1;
+    uint8 public constant QT_ETH_CALL_BY_TIMESTAMP = 2;
+    uint8 public constant QT_ETH_CALL_WITH_FINALITY = 3;
+    uint8 public constant QT_ETH_CALL_MAX = 4; // Keep this last
```

Comment:
should this be named `QT_MAX` since it's related solely to query type, not eth call?